upgrade/1.2/Stage_0_Prerequisites.md

```diff
@@ -2,50 +2,56 @@
 
 > **NOTE:** CSM-1.0.1 or higher is required in order to upgrade to CSM-1.2.0
 
-* [Stage 0.1 - Install latest docs RPM](#install-latest-docs)
-* [Stage 0.2 - Reduce CPU Limits If Needed](#reduce-cpu-limits)
-* [Stage 0.3 - Update SLS](#update-sls)
-* [Stage 0.4 - Upgrade Management Network](#update-management-network)
-* [Stage 0.5 - Prerequisites Check](#prerequisites-check)
-* [Stage 0.6 - Backup VCS Data](#backup-vcs-data)
-* [Stage 0.7 - Suspend NCN Configuration](#suspend-ncn-config)
-* [Stage 0.8 - Backup Workload Manager Data](#backup_workload_manager)
-* [Stage 0.9 - Modify NCN Images](#modify_ncn_images)
-* [Stage 0.10 - Continue to Stage 1](#continue_to_stage1)
+- [Stage 0.1 - Install latest docs RPM](#install-latest-docs)
+- [Stage 0.2 - Reduce CPU Limits If Needed](#reduce-cpu-limits)
+- [Stage 0.3 - Update SLS](#update-sls)
+- [Stage 0.4 - Upgrade Management Network](#update-management-network)
+- [Stage 0.5 - Prerequisites Check](#prerequisites-check)
+- [Stage 0.6 - Backup Workload Manager Data](#backup_workload_manager)
+- [Stage 0.7 - Continue to Stage 1](#continue_to_stage1)
 
 <a name="install-latest-docs"></a>
+
 ## Stage 0.1 - Install latest docs RPM
 
-1. Install latest document RPM package:
+1. Install latest document RPM package and prepare assets:
+
+   > The install scripts will look for the RPM in `/root`, so it is important that you copy it there.
 
-    > The install scripts will look for the RPM in `/root`, so it is important that you copy it there.
+   - Internet Connected
 
-    * Internet Connected
+     ```bash
+     wget https://storage.googleapis.com/csm-release-public/csm-1.2/docs-csm/docs-csm-latest.noarch.rpm -P /root
+     
+     rpm -Uvh --force /root/docs-csm-latest.noarch.rpm
 
-        ```bash
-        ncn-m001# wget https://storage.googleapis.com/csm-release-public/csm-1.2/docs-csm/docs-csm-latest.noarch.rpm -P /root
-        ncn-m001# rpm -Uvh --force /root/docs-csm-latest.noarch.rpm
-        ```
+     /usr/share/doc/csm/upgrade/1.2/scripts/upgrade/prepare-assets.sh --csm-version [CSM_RELEASE] --endpoint [ENDPOINT]
+     ```
 
-    * Air Gapped (replace the PATH_TO below with the location of the rpm)
+   - Air Gapped (replace the PATH_TO below with the location of the rpm)
```

Comment:
It looks like the only change here is the indentation and changing the asterisk to a dash. It looks like there are a lot of places in the file where indentation was changed by one space like this. Unless this is fixing a problem, I advise against it, just because it makes the PR much harder to review (since many lines show up as being different when in fact this is the only change)